src/robot_dart/gui/magnum/base_application.hpp

```diff
@@ -131,7 +131,9 @@ namespace robot_dart {
                 void update_lights(const gs::Camera& camera);
                 void update_graphics();
                 void render_shadows();
+                // This should not be used
```

Comment:
why?

Reply:
Because the function with just the name would stop on the first `BodyNode` or `ShapeNode` with the given name. This is OK if you have only one robot in the world or all the body nodes/shape nodes have different names. This is not always the case, since body nodes and shape nodes are allowed to have the same name if they are under different skeletons (robots). I meant to delete this function as I created one with a pointer to the body node; I forgot. I will delete it.